.github/workflows/pre_release.yml

```diff
@@ -0,0 +1,49 @@
+name: pre-release
+
+on:
+  create
+
+jobs:
+
+  # Make a PR to master from a new branch with changes, and delete the created one
+  update-versions:
+
+    # Only run when a release-v* branch is created, and not by drashbot
+    if: contains(github.ref, 'release-v') && !contains(github.event.sender.login, 'drashbot')
+
+    strategy:
+      matrix:
+        deno: ["1.5.1"]
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Checkout master
+        uses: actions/checkout@v2
+        with:
+          ref: master
+
+      - name: Delete Remote Release Branch
```

Comment:
comment out?

Reply:
good catch, i swear down i commented that out :S

Reply:
https://github.com/drashland/sockets/pull/95/commits/e257cb80032e672c15e1164888981cacc39cf81e